From a51f887042419d721fcb996751b74e528dbabaf5 Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Thu, 22 Sep 2016 06:28:25 +0200
Subject: Fix appdata installation

Add back @APPSTREAM_XML_RULES@ that got accidentally lost in commit
8ae85f0.
---
 editor/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/editor/Makefile.am b/editor/Makefile.am
index 3ef2952..a3d13c1 100644
--- editor/Makefile.am
+++ editor/Makefile.am
@@ -68,6 +68,7 @@ ca.desrt.dconf-editor.service: Makefile
 	             echo 'Exec=${bindir}/dconf-editor --gapplication-service') > $@.tmp && \
 	             mv $@.tmp $@
 
+@APPSTREAM_XML_RULES@
 appstream_in_files = ca.desrt.dconf-editor.appdata.xml.in
 appstream_XML = $(appstream_in_files:.xml.in=.xml)
 @INTLTOOL_XML_RULE@
-- 
cgit v0.12

